core/src/main/java/nl/nn/adapterframework/senders/XsltSender.java

```diff
@@ -487,6 +382,10 @@ public void setOmitXmlDeclaration(boolean b) {
 		omitXmlDeclaration = b;
 	}
 	public boolean isOmitXmlDeclaration() {
+		Boolean omitXmlDeclaration=getOmitXmlDeclaration();
+		return omitXmlDeclaration!=null?omitXmlDeclaration:true;
+	}
+	public Boolean getOmitXmlDeclaration() { // can return null too
```

Comment:
Moet dit null terug kunnen geven? Waarom twee getters?

Reply:
Dit moet null terug kunnen geven, om aan te geven dat er niets door middel van attributen geconfigureerd is. In dat geval wordt gekeken of er een waarde in de xslt geconfigureerd is. Als die er ook niet is wordt pas een default gebruikt.